descriptors/ModuleDescriptor-template.json

```diff
@@ -1013,7 +1013,7 @@
     },
     {
       "id": "feesfines",
-      "version": "15.0 16.0"
+      "version": "15.0 16.1"
```

Comment:
Why is version 16.1 specifically needed?

Reply:
Because the new property `cancellationReason` will be available since `16.1`, see https://github.com/folio-org/mod-feesfines/pull/143

Reply:
I think that means that dual support with 15.x and 16.1 is not possible, as no 15.x version has that new property.

Reply:
Yes, agree. Should I remove the 15.x? support?

Reply:
Yes, I think we need to

Reply:
Ok, done.